includes/wikia/services/ApiService.class.php

```diff
@@ -95,18 +90,20 @@ static function foreignCall( $dbName, Array $params, $endpoint = self::API, $set
 	 *
 	 * @return string HTTP domain
 	 */
-	private static function getHostByDbName( $dbName ) {
+	private static function getHostByDbName( string $dbName ) {
 		global $wgDevelEnvironment, $wgDevDomain;
 
-		$cityId = WikiFactory::DBtoID( $dbName );
-		$hostName = WikiFactory::getVarValueByName( 'wgServer', $cityId );
-
 		if ( !empty( $wgDevelEnvironment ) ) {
+			$hostName = WikiFactory::DBtoUrl( $dbName );
```

Comment:
Why do we want to use `DBtoUrl` on devboxes and `wgServer` on the rest of envs? If there is a good reason for that, please add a comment to the code.